docs/guides/react-components/install-plain-java.md

```diff
@@ -22,7 +22,7 @@ Confirm Elasticsearch is running by visiting `http://localhost:9200` in a browse
 
 ### Download and configure Camunda
 
-1. Download the [latest release artifact](https://github.com/camunda/camunda/releases) in the **Assets** section of the release page, starting with [8.6.0-alpha2](https://github.com/camunda/camunda/releases/tag/8.6.0-alpha2).
+1. Download and extract the [latest `camunda-zeebe-` release artifact](https://github.com/camunda/camunda/releases) in the **Assets** section of the release page, starting with [8.6.0-alpha2](https://github.com/camunda/camunda/releases/tag/8.6.0-alpha2).
```

Comment:
Note that this change is only added in vNext, because vCurrent uses the docker-compose installation method (nothing to extract).